lib/retrieve-submission-data.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+'use strict'
+
+/* ::
+import type {
+  RetrievalData
+} from '../types.js'
+*/
+
+const axios = require('axios')
+const AWS = require('aws-sdk')
+
+function getCredentials (
+  formId /* : string */,
+  submissionId /* : string */,
+  jwt /* : string */
+) {
+  let api = axios.create({
+    baseURL: 'https://auth-api-test.blinkm.io',
+    headers: {'Authorization': `Bearer ${jwt}`}
+  })
+
+  return api.get(`/forms/${formId}/retrieval-credentials/${submissionId}`)
+    .then(function (response) {
+      return response.data
+    })
+}
+
+function getDataFromS3 (
+  retrievalData /* : RetrievalData */
+) {
+  const s3 = new AWS.S3({
+    region: 'ap-southeast-2',
```

Comment:
We should add this to the API so that the SDK can just use `retrievalData.s3.region`

Reply:
The API has been updated to return this value now.